### PR TITLE
test-netlink: Make sure netinet/in.h is not include sorted

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

File: src/libsystemd/sd-netlink/test-netlink.c [hunk 1]
```diff
@@ -1,13 +1,16 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <net/if.h>
-#include <netinet/ether.h>
+/* We make sure netinet/in.h is not include sorted automatically by clang-format because it will happily
+ * redefine in_addr even if it has already been defined by the linux headers. */
 #include <netinet/in.h>
```
Comment (keszybz) [inline src/libsystemd/sd-netlink/test-netlink.c:5]: How does this work? If there's an empty line, clang-format doesn't reorder across it?

Comment (daandemeyer) [reply src/libsystemd/sd-netlink/test-netlink.c:5]: Exactly

**Comment by yuwata** [comment]
We already do like that in many places, especially, src/network. If you touch it, please do more consistently.

**Comment by DaanDeMeyer** [comment]
I've opened https://sourceware.org/bugzilla/show_bug.cgi?id=29354 to hopefully get the bug fixed in glibc instead.

**Pull request closed without merging**